guides/2022-04-05-conduit-kafka-connect-connector.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: "Using Kafka Connect Connectors with Conduit"
+slug: using-kafka-connect-connectors-with-conduit
+author: '@anaptfox'
+author_url: https://twitter.com/anaptfox
+author_image_url: https://pbs.twimg.com/profile_images/1004707739405205504/BpvzOr9X_400x400.jpg
+tag: ['File Connector']
+description: Learn how to build a Conduit Algolia Connector
+keywords: ['data', 'connector', 'algolia', 'conduit']
+---
+
+# Using Kafka Connect Connectors with Conduit
+
+The Conduit Kafka Connect Wapper connector is a special connector that allows you to use Kafka Connect connectors with Conduit. 
+
+This gives you the ability to: 
+- Use a datastore if Conduit doesn't have a native connector.
+- Remove Kafka as a dependency to simply move data from one place to another.
+- Easily migrate from Kafka Connect to Conduit.
+
+Since the Conduit Kafka Connect Wapper itself is written in Java, but most of Conduit's connectors are written in Golang, it also serves as a good example of the flexbilty of the Conduit Plugin SDK.
+
+At a high level, to use the Kafka Connect Wapper, you'll need to:
+
+1. Download the Wapper Connector.
+1. Build the Connector.
+1. Download the Kafka Connect JARs and any dependencies.
+1. Create Pipeline
+1. Add the Kafka Connect Connector to Pipeline.
+
+Let's begin.
+
+<!--truncate-->
+
+## Setup
+
+To begin, let's run the setup script.
+
+```
+./scripts/dist.sh
+```
+
+This script will creates a directory called `dist` with following contents:
+
+1. The connector JAR itself
+1. Directory `libs` with Aiven's JDBC connector (but no JDBC drivers). 
+
+The `libs` directory is where you put the Kafka Connect connector JARs and their dependencies (if any). The wapper plugin will automatically load connectors and all the other dependencies from a `libs` directory.
+
+## Adding Kafka Connect Connector
+
+Now that we have everything setup, we can add the Kafka Connect JARS. For this example, we will use the PostgreSQL Kafka Connect JDBC Connector. To install, add the following:
+
+- Add JDBC JAR
```

Comment:
We already include Aiven's JDBC connector, so this can be omitted.

Reply:
After the last standup seems like that's not going to be the case anymore, can this comment be resolved @hariso?

Reply:
Yes, that's correct, we provide no connectors out of the box anymore.